klasses/api_test.py

```diff
@@ -289,41 +320,44 @@ def test_defer_enrollment(mocker, user, force):
         force=force,
     )
     assert returned_from_enrollment == patched_deactivate_enrollments.return_value
-    assert returned_to_enrollment == patched_create_enrollments.return_value[0]
-    patched_create_enrollments.assert_called_once_with(
-        existing_enrollment.user, [target_run], order=order
+    assert returned_to_enrollment == patched_create_enrollment.return_value
+    patched_create_enrollment.assert_called_once_with(
+        existing_enrollment.user, target_run, order=order
     )
     assert patched_deactivate_enrollments.call_count == 1
     assert patched_deactivate_enrollments.call_args == mocker.call(
         change_status=ENROLL_CHANGE_STATUS_DEFERRED, run_enrollment=existing_enrollment
     )
 
 
-@pytest.mark.django_db
-def test_defer_enrollment_same_bootcamp_run_validation(
-    enrollment_data,
-    patched_create_run_enrollments,
-    patched_deactivate_run_enrollment,
-    user,
-):
-    """
-    defer_enrollment should raise an exception if the 'from' or 'to' bootcamp runs are same
-    """
-    with pytest.raises(ValidationError):
-        # Deferring to the same bootcamp run should raise a validation error
-        defer_enrollment(
-            user,
-            enrollment_data.enrollments[0].bootcamp_run.bootcamp_run_id,
-            enrollment_data.enrollments[0].bootcamp_run.bootcamp_run_id,
-            enrollment_data.order.id,
-        )
-    patched_create_run_enrollments.assert_not_called()
+@pytest.fixture()
+def enrollment_data(user):
```

Comment:
Nit: our current pattern is to define fixtures before all test cases in the file. This includes `patched_create_run_enrollment` etc.